src/utils/azureUtils.ts

```diff
@@ -1,28 +1,10 @@
 /*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
+*  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Something stole these spaces

Reply:
Side note: the spaces here on the header aren't consistent across our files